platform/ios/test/MGLMapViewConversionTests.m

```diff
@@ -0,0 +1,52 @@
+#import <Mapbox/Mapbox.h>
+#import <XCTest/XCTest.h>
+
+#import "../../ios/src/MGLMapView.h"
+
+@interface MGLMapViewConversionTests : XCTestCase
+
+@property (nonatomic) UIView *superView;
+@property (nonatomic) MGLMapView *mapView;
+
+@end
+
+@implementation MGLMapViewConversionTests
+
+- (void)setUp {
+    [super setUp];
+    
+    [MGLAccountManager setAccessToken:@"pk.feedcafedeadbeefbadebede"];
+    self.superView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, 64, 64)];
+    self.mapView = [[MGLMapView alloc] initWithFrame:CGRectMake(0, 0, 64, 64) styleURL:MGLStyle.streetsStyleURL];
+    [self.mapView setCenterCoordinate:CLLocationCoordinate2DMake(33, 179)];
+    
+    [self.superView addSubview:self.mapView];
```

Comment:
Does this test pass if no superview is involved? If it’s possible to remove the reference to UIView from this file, then it can be shared between the iOS and macOS test targets in the platform/darwin/ folder.